tools/terraform/networking.tf

```diff
@@ -67,26 +67,42 @@ resource "google_compute_url_map" "global" {
   description     = "a description"
   default_service = google_compute_backend_service.global.id
 
+
   // Create a host rule to match traffic to alias (gcpping.com)
+  host_rule {
+    hosts        = ["*"]
+    path_matcher = "endpoints-config-bucket"
+  }
+
   dynamic "host_rule" {
     for_each = var.domain_alias_flag ? [1] : []
 
     content {
       hosts = [
         var.domain_alias,
+        "*.${var.domain_alias}",
```

Comment:
Found a bug while testing redirects, we were only redirect the alias for the naked domain, but not for subdomains of the alias.

Before: 
`gcpping.com` > `gcping.com`
`www.gcpping.com` > `www.gcpping.com`

After: 
`gcpping.com` > `gcping.com`
`www.gcpping.com` > `gcping.com`